istioctl/cmd/istioctl/version_test.go

```diff
@@ -197,6 +197,6 @@ func (client mockExecVersionConfig) GetIstioVersions(namespace string) (*version
 	return &meshInfo, nil
 }
 
-func mockExecClientVersionTest(kubeconfig, configContext string) (kubernetes.ExecClient, error) {
+func mockExecClientVersionTest(_, _ string) (kubernetes.ExecClient, error) {
```

Comment:
Why do we want to remove unused parameter names from the code?  Aren't they considered internal documentation?

Reply:
In general, that's against Go philosophy. In general, code is as brief as it can be. This is why for example, most variables are declared with:

   x := MyFunc()

instead of:

  var x string
  x = MyFunc